tools/documentation/_includes/meta-info.njk

```diff
@@ -11,31 +11,7 @@
 <link rel="stylesheet" type="text/css" href="/components/vars/spectrum-large.css"/>
 <link rel="stylesheet" type="text/css" href="/components/tokens/index.css"/>
 
-{% for dep in [
-  'accordion',
-  'actionbutton',
-  'button',
-  'card',
-  'clearbutton',
-  'divider',
-  'fieldlabel',
-  'icon',
-  'illustratedmessage',
-  'link',
-  'menu',
-  'page',
-  'picker',
-  'popover',
-  'search',
-  'sidenav',
-  'site',
-  'statuslight',
-  'table',
-  'textfield',
-  'tooltip',
-  'typography',
-  'underlay'
-] %}
+{% for dep in components %}
```

Comment:
I think this is a good patch for now. Originally that list of components was curated to match the styles being used in the website rendering separate from the documentation pages for the individual components. For a first pass, loading all the dependencies will work but I think eventually, we'll want to load components more minimally based on what's being used.

Reply:
Yea but we need to do a thorough investigation on this since there are many adhoc component css are loaded in every pages.